src/reducers/MapReducer.js

```diff
@@ -164,13 +134,20 @@ export default (state = INITIAL_STATE, action) => {
         case INITIALIZE_MAP:
             const {region, sites} = payload;
             const existingMapLoadedState = state.mapLoaded;
+            const selectedSiteIsEmpty = _.isEmpty(state.selectedSite);
+            const lastKnownRegionIncludingSelectedSite = selectedSiteIsEmpty ? region : {
+                longitudeDelta: 0.25,
+                latitudeDelta: 0.25,
+                longitude: state.selectedSite.coordinate.longitude,
+                latitude: state.selectedSite.coordinate.latitude
```

Comment:
This is the work for GH-76; if we view a site from the ListView in `SiteDetailScreen` and then go back to the `MapView`, let's zero in on that site.